contrib/NeonVM/neonvm.c

```diff
@@ -0,0 +1,820 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "opcode.h"
+#include "stack.h"
+
+#define error(s) fprintf(stderr, "%s\n", s); abort();

```

Comment:
This would be better as a function. As written, this macro is wrong. (See https://stackoverflow.com/questions/257418/do-while-0-what-is-it-good-for for details.)